www/src/components/docs-breadcrumb.js

```diff
@@ -0,0 +1,90 @@
+import React from "react"
+import { Link } from "gatsby"
+import getActiveItem from "../utils/sidebar/get-active-item"
+import getActiveItemParents from "../utils/sidebar/get-active-item-parents"
+import { mediaQueries, space, fontSizes } from "../utils/presets"
+import { rhythm } from "../utils/typography"
+
+const Separator = ({ character = `>` }) => (
```

Comment:
I have to say I dislike using `>` (and `<`); they feel clunky, IMO they should be less wide. Maybe we can try a chevron (`react-icons/lib/md/chevron-right`)?

Reply:
I think that would be cool. What's your thought? @gillkyle

Reply:
That sounds good to me! I actually considered leaving feedback about the separator character, but didn't feel as strongly about it. Using a chevron would be slightly more sophisticated though.

Reply:
Sounds like a great idea! Been swamped today so I could get to it later or if @mikeattara wants to take a stab at it that'd be great

Reply:
Looks a lot better that way: 
![image](https://user-images.githubusercontent.com/21114044/60469123-dd739b00-9c18-11e9-924b-8bc5015dfc36.png)